core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -260,7 +260,8 @@ private[spark] object TestUtils {
       contains = contain(e, msg)
     }
     assert(contains,
-      s"Exception tree doesn't contain the expected exception ${typeMsg}with message: $msg")
+      s"Exception tree doesn't contain the expected exception ${typeMsg}with message: $msg\n" +
+        Utils.exceptionString(e))
```

Comment:
I found this small adjustment really useful when using `assertExceptionMsg`, to be able to see the actual output when the expectation doesn't match.